### PR TITLE
[tests-only] Use owncloudci/wait-for docker image

### PR DESCRIPTION
**Pull request by phil-davis**
Update the drone starlark code the same as https://github.com/owncloud/activity/pull/1036

### CODE REVIEW

**Comment by phil-davis** [comment]
wait-for elastic search works:
https://drone.owncloud.com/owncloud/search_elastic/1860/19/10
```
latest: Pulling from owncloudci/wait-for
Digest: sha256:08aa5a5f16c7997f37568efc087b4501afea851cfd0143d898e64e0eeb0310e3
Status: Image is up to date for owncloudci/wait-for:latest
+ wait-for -it elasticsearch:9200 -t 600
services are ready!
```

File: .drone.star [hunk 1]
```diff
@@ -309,7 +314,7 @@ def phpstan(ctx):
         return pipelines
 
     default = {
-        "phpVersions": ["7.4"],
+        "phpVersions": ["7.3"],
```
Comment (kiranparajuli589) [inline .drone.star:317]: why degrading to `7.3` :thinking: ?

Comment (phil-davis) [reply .drone.star:317]: Hmm - that's an accident!

Comment (phil-davis) [reply .drone.star]: fixed

**Review by kiranparajuli589 — APPROVED** [review]

**Pull request merged**